sentry-core/src/main/java/io/sentry/core/transport/StdoutTransport.java

```diff
@@ -0,0 +1,30 @@
+package io.sentry.core.transport;
+
+import io.sentry.core.ISerializer;
+import io.sentry.core.SentryEvent;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.io.PrintWriter;
+import java.io.Writer;
+import java.nio.charset.Charset;
+
+public final class StdoutTransport implements ITransport {
+  private final ISerializer serializer;
+
+  public StdoutTransport(final ISerializer serializer) {
+    this.serializer = serializer;
+  }
+
+  @Override
+  public TransportResult send(SentryEvent event) throws IOException {
+    Writer outputStreamWriter = new OutputStreamWriter(System.out, Charset.defaultCharset());
```

Comment:
I'd recommend `Charset.forName("UTF-8")` instead of `Charset.defaultCharset()` as we always use UTF8, so it's good to keep compatibility.

Reply:
Done. Thank you, wasn't sure what to do there.